CHANGELOG.adoc

```diff
@@ -11,6 +11,40 @@
 
 === Breaking changes
 
+- https://github.com/eclipse-sirius/sirius-web/issues/2658[#2658] [sirius-web] Add dedicated colors for navigation bar in the Sirius Web theme.
+The theme provided by the user must now provide navigation bar colors:
+```ts
+...
+navigationBar: {
+  border: 'yellow',
+  background: 'blue',
+},
+...
+```
+- https://github.com/eclipse-sirius/sirius-web/issues/2669[#2669] [sirius-web] Add dedicated colors for selected diagram elements in the Sirius Web theme.
+The theme provided by the user must now provide selected diagram elements colors:
+```ts
+...
+diagram: {
+  selectedEdge: 'green',
+  selectedNode: 'red',
+},
+...
+```
+- https://github.com/eclipse-sirius/sirius-web/issues/2672[#2672] [sirius-web] Add dedicated colors for toast messages in Sirius Web theme.
+The theme provided by the user must now provide toast messages colors:
+```ts
+...
+toast: {
+  default: '#7269A4',
+  success: '#43a047',
+  error: '#d32f2f',
+  warning: '#ff9800',
+  info: '#2196f3',
+},
+...
```

Comment:
The various severity levels are configurable with the values `theme.palette.success`, `theme.palette.error`, `theme.palette.warning`, `theme.palette.info` for all components (for example, the inline validation errors of the form view). I don't see why we would want different colors for the severity in different location. This should work for the toast component too.